src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationValidService.java

```diff
@@ -18,6 +24,23 @@ static final boolean isMandatoryLetter(NotificationEventType eventType) {
         return STRUCK_OUT.equals(eventType);
     }
 
+    static final boolean isFallbackLetterRequiredForSubscriptionType(NotificationWrapper wrapper, SubscriptionType subscriptionType, NotificationEventType eventType) {
```

Comment:
the final special word in the static method is not needed.

Reply:
Fixed in next push.